Makefile

```diff
@@ -294,12 +297,12 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
-	$(OPERATOR_SDK) bundle validate ./bundle
+	$(OPERATOR_SDK) bundle validate ./bundle --select-optional suite=operatorframework --optional-values=k8s-version=${VALIDATE_K8S_VERSION}
 
 .PHONY: bundle-k8s
 bundle-k8s: bundle # Generate bundle manifests and metadata for Kubernetes, then validate generated files.
 	$(KUSTOMIZE) build config/manifests-k8s | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
-	$(OPERATOR_SDK) bundle validate ./bundle
+	$(OPERATOR_SDK) bundle validate ./bundle --select-optional suite=operatorframework --optional-values=k8s-version=${VALIDATE_K8S_VERSION}
```

Comment:
IIUC the change is now that validation is done according to a specific k8s version.
I wonder how did it work before this change ? (i.e how was the version for the validation chosen)

Reply:
Beforehand there was no Kubernetes deprecated APIs check/validation. 
Having said that, I still don't understand the effect of `--optional-values=k8s-version`  option. I have tested NMO (SNR and NHC as well) against this validation with and without --optional-values=k8s-version and the results were similar.

Reply:
So I'm completely clear are we testing with Kubernetes deprecated APIs in addition to what we've previously tested or instead of it ?

Reply:
By using ` --select-optional suite=operatorframework`, we are adding new vaildators for the bundle validation.

Reply:
The similar results I have seen in the past (regardless of the `--optional-values=k8s-version` flag), were on purpose since the bundle validation doesn't read code, then it can't verify the version which we are using for the deprecated APIs. If it can't verify, then it will warn (and won't error).